src/main/java/mil/dds/anet/beans/AttachmentClassification.java

```diff
@@ -0,0 +1,30 @@
+package mil.dds.anet.beans;
+
+import io.leangen.graphql.annotations.GraphQLInputField;
+import io.leangen.graphql.annotations.GraphQLQuery;
+import mil.dds.anet.views.AbstractAnetBean;
+
+public class AttachmentClassification extends AbstractAnetBean {
+
+  public static enum Classification {
```

Comment:
Why is this in a separate class, and not simply an enum inside the attachment bean?